.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Ask a question
+    url: https://forums.aws.amazon.com/
```

Comment:
what's the reasoning behind redirecting to the AWS Forums for questions?

Reply:
Updated to the `amazon-sagemaker` page on Stack Overflow.